src/tests/tests_general.jl

```diff
@@ -272,6 +276,16 @@ function test_manifold(
 
         Test.@test norm(M, pts[1], X1) isa Real
         Test.@test norm(M, pts[1], X1) ≈ sqrt(inner(M, pts[1], X1, X1))
+
+        (test_inplace && is_mutating) && Test.@testset "inplace test for exp!" begin
+            p = deepcopy(pts[1])
+            X = deepcopy(X1)
+            q = exp(M, p, X)
+            exp!(M, p, p, X)
+            Test.@test p == q
```

Comment:
Why not `isapprox(M, p, q)`?

Reply:
Right, I might change that. My first idea was, that they really should be equal.

Reply:
They should but I wouldn't say that they have to be. `exp` and `exp!` could potentially use slightly different code that leads to different numerical errors.